diff --git a/node_modules/react-native-leveldb/android/build.gradle b/node_modules/react-native-leveldb/android/build.gradle
index c263715..3ddd30c 100644
--- a/node_modules/react-native-leveldb/android/build.gradle
+++ b/node_modules/react-native-leveldb/android/build.gradle
@@ -1,16 +1,3 @@
-buildscript {
-  repositories {
-    maven {
-      url 'https://plugins.gradle.org/m2/'
-    }
-    mavenCentral()
-    google()
-  }
-
-  dependencies {
-    classpath('com.android.tools.build:gradle:7.2.2')
-  }
-}
-
 def resolveBuildType() {
   Gradle gradle = getGradle()
   String tskReqStr = gradle.getStartParameter().getTaskRequests()['args'].toString()
@@ -41,6 +28,7 @@ def reactNativeArchitectures() {
 }
 
 repositories {
+  google()
   mavenCentral()
 }
